### PR TITLE
Add aws_elb and aws_elbs resources

### PR DESCRIPTION
**Pull request by clintoncwolfe**
Adds `aws_elb` singular and `aws_elbs` plural resources.  Fairly straightforward if long-winded.

One change of note: I adjusted the way CIDRs are calculated for subnets in the Terraform code, to be more flexible.

Refs #2826

### CODE REVIEW

**Review by jquick — APPROVED** [review]
Nice additions @clintoncwolfe !

File: docs/resources/aws_elbs.md.erb [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+Returns the FQDN of the load balancer.  This is the hostname which is exposed to the world.
+
+    # Find ELBs that have the letter z in their DNS name
+    describe aws_elbs.where(dns_name: /z/) do
+      it { should exist }
+    end
+
+### elb\_name
+
+The name of the ELB within AWS. The ELB name is unique within the region. If you know the full ELB name, you should use the `aws_elb` resource instead, as it is much more efficient.
```
Comment (jerryaldrichiii) [inline docs/resources/aws_elbs.md.erb]: Could we change this to say `much more efficient for testing a specific ELB`?

File: test/integration/aws/default/build/ec2.tf [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Contains resources and outputs related to testing the aws_ec2_* resources.
 
+# This is the
```
Comment (jerryaldrichiii) [inline test/integration/aws/default/build/ec2.tf]: It is the? :smile:

**Review by jerryaldrichiii — CHANGES_REQUESTED** [review]
Looks good to me! Just a few minor things.

File: test/integration/aws/default/build/ec2.tf (hunk 2 above)
Comment (clintoncwolfe) [reply test/integration/aws/default/build/ec2.tf]: Deleted - I have no idea what that was!

**Review by jerryaldrichiii — APPROVED** [review]
Looks great! Thanks @clintoncwolfe.

**Pull request merged**